src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -425,7 +425,7 @@
     </ItemGroup>
     <ItemGroup>
       <ReferencePath Condition="
-              '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime'
+              '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime'
               or '%(FileName)' == 'Microsoft.Internal.VisualStudio.Shell.Interop.14.0.DesignTime'
```

Comment:
I think the MS.Internal.VS.Shell.Interop.14.0.DesignTime name is bogus, can you remove it?